client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -370,7 +376,9 @@ const ConnectedSiteSetupList = connect( ( state, props ) => {
 		menusUrl: getCustomizerUrl( state, siteId, null, null, 'add-menu' ),
 		siteId,
 		siteSlug: getSiteSlug( state, siteId ),
-		tasks: taskList.getAll(),
+		tasks: taskList.getAllSorted(
+			unverifiedEmailTaskComparator( ! isCurrentUserEmailVerified( state ) )
```

Comment:
Could we move `! isCurrentUserEmailVerified( state )` to a variable to use in this argument and in the `isEmailUnverified` property?

Reply:
Good suggestion, I've moved it to a variable here https://github.com/Automattic/wp-calypso/pull/84291/commits/ac055ce553473bbca6d2221500486a59348e3173